File: app/src/main/java/me/zipi/navitotesla/model/Vehicle.java
package me.zipi.navitotesla.model;


import com.google.gson.annotations.SerializedName;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@NoArgsConstructor
@AllArgsConstructor
@Data
public class Vehicle {

    Long id;
    @SerializedName("vehicle_id")
    Long vehicleId;

    @SerializedName("display_name")
    String displayName;

    String state;

}

File: app/src/main/kotlin/me/zipi/navitotesla/model/Vehicle.kt
package me.zipi.navitotesla.model

import com.google.gson.annotations.SerializedName

data class Vehicle(
    val id: Long,
    @SerializedName("vehicle_id")
    val vehicleId: Long,
    @SerializedName("display_name")
    val displayName: String,
    val state: String,
)
